lib/email/styles.rb

```diff
@@ -180,7 +180,7 @@ def format_html
       correct_footer_style_hilight_first
       reset_tables
 
-      html_lang = SiteSetting.default_locale.sub("_", "-")
+      html_lang = I18n.locale.to_s.sub("_", "-")
       style('html', nil, lang: html_lang, 'xml:lang' => html_lang)
       style('body', "text-align:#{ Rtl.new(nil).enabled? ? 'right' : 'left' };")
       style('body', nil, dir: Rtl.new(nil).enabled? ? 'rtl' : 'ltr')
```

Comment:
After this change `Rtl.new(nil).enabled?` might return a wrong value because it doesn't use `I18n.locale`